test/javaee6/viewParamNullValueAjax/src/main/java/com/sun/faces/test/javaee6/viewParamNullValueAjax/TestView.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 1997, 2018 Oracle and/or its affiliates. All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0, which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception, which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ */
+
+package com.sun.faces.test.javaee6.viewParamNullValueAjax;
+
+import java.io.Serializable;
+
+import javax.enterprise.inject.Model;
+import javax.faces.view.ViewScoped;
+import javax.inject.Named;
+
+@Model
```

Comment:
Remove this. It's a stereotype for `@Named @RequestScoped`. But it's being overridden with `@ViewScoped @Named` in following lines.

Reply:
Thank you for the review, I rebased and removed that annotation.